core/io/resource_importer.cpp

```diff
@@ -137,6 +137,17 @@ Ref<Resource> ResourceFormatImporter::load(const String &p_path, const String &p
 	}
 
 	Ref<Resource> res = ResourceLoader::_load(pat.path, p_path, pat.type, p_cache_mode, r_error, p_use_sub_threads, r_progress);
+	if (res.is_null()) {
+		// Filename has already been printed in _load()
```

Comment:
```suggestion
		// Filename has already been printed in _load().
```
Comment style nitpick